### PR TITLE
ART-1062 adding find-builds suport attach multiple images with different pv

### PR DESCRIPTION
**Pull request by shiywang**
an alternative solution for https://github.com/openshift/elliott/pull/60 without touching errata-tool

### CODE REVIEW

**Comment by tbielawa** [comment]
How can I test this? Example commands would be useful

**Comment by sosiouxme** [comment]
Seems to work. I tried it out on real data (after removing two test subjects) like so:
```
🐚 elliott -g openshift-4.2 find-builds --kind image 
...
The following 2 builds may be attached to an advisory:
 ironic-inspector-container-v4.2.1-201910171309
 jenkins-agent-maven-35-rhel7-container-v4.2.1-201910171309

🐚 elliott -g openshift-4.2 find-builds --kind image --use-default-advisory image
...
Attached build(s) successfully:
 ironic-inspector-container-v4.2.1-201910171309
 jenkins-agent-maven-35-rhel7-container-v4.2.1-201910171309
```

File: elliottlib/model.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-class ModelException(StandardError):
+class ModelException(Exception):
```
Comment (sosiouxme) [inline elliottlib/model.py]: ... why?

Comment (shiywang) [reply elliottlib/model.py]: oh, this change was made by `autopep8` I use it to pass CI
the exception hierarchy tree is sth like this:
```
BaseException
 +-- SystemExit
 +-- KeyboardInterrupt
 +-- GeneratorExit
 +-- Exception
      +-- StopIteration
      +-- StandardError
```
So I guess `autopep8` decided to switch to `Exception` to help us catch more exceptions but base on our code is not that robust, I can change it back.

**Comment by shiywang** [comment]
@sosiouxme @tbielawa  I guess for adding new commands or new options like `--from-diff` we should definitely add examples, this change it more insensible because I didn't introduce images and either bring confusing into the existing behavior of `elliott find-builds`

**Pull request merged**